components/app/manage/project/ProjectDescription.vue

```diff
@@ -31,6 +31,7 @@ import AppFormControl from '@/components/common/AppFormControl';
 import AppTextarea from '@/components/common/AppTextarea';
 import ProjectConstants from '@/mixins/ProjectConstants';
 
+/** @deprecated can be replaced with project/TextArea */
```

Comment:
question(non-blocking): how much work is this? Can we do it in this pr?

Reply:
Replaced and removed legacy component